### PR TITLE
Modified Shield, added Paddle Boost, Enlarge, Inverted, Slow and Shrink

### PR DESCRIPTION
**Pull request by JeromeTiong**
@chesterhow @ivantjh

### CODE REVIEW

File: src/graphics.h [hunk 1]
```diff
@@ -39,6 +39,7 @@ namespace graphicsNS {
 	const COLOR_ARGB BLUE = D3DCOLOR_ARGB(255, 84, 195, 255);
 	const COLOR_ARGB ORANGE = D3DCOLOR_ARGB(255, 249, 162, 50);
 	const COLOR_ARGB RED = D3DCOLOR_ARGB(255, 255, 105, 97);
+	const COLOR_ARGB YELLOW = D3DCOLOR_ARGB(255, 255, 255, 100);
```
Comment (chesterhow) [inline src/graphics.h]: Dunnid this anymore

**Comment by JeromeTiong** [comment]
fixes #27 #28 #29 #30 #32

File: src/paddle.cpp [hunk 2]
```diff
@@ -64,23 +136,28 @@ void Paddle::runEffects() {
 				case effectNS::ENLARGE: {
 					spriteData.scale = (currentEffect.second == 0) ? 1.0f : 2.0f;
 				} break;
-				case effectNS::INVERT: {			
+				case effectNS::SHRINK: {
+					spriteData.scale = (currentEffect.second == 0) ? 1.0f : 0.5f;
+				} break;
+				case effectNS::INVERT: {	
+					inverted = (currentEffect.second == 0) ? false : true;
```
Comment (chesterhow) [inline src/paddle.cpp]: No need for ternary operator here
```
inverted = currentEffect.second != 0;
```

File: src/paddle.cpp [hunk 3]
```diff
@@ -64,23 +136,28 @@ void Paddle::runEffects() {
...
 					msgPtr = new Message(messageNS::RUN_EFFECT, messageNS::BALL, effectNS::SHIELD, id);
 					setMessage(msgPtr);
 				} break;
-				case effectNS::MULTIPLY: {
-				} break;
 				case effectNS::BOOST: {
+					boosted = (currentEffect.second == 0) ? false : true;
 				} break;
 				case effectNS::SLOW: {
+					slowed = (currentEffect.second == 0) ? false : true;
```
Comment (chesterhow) [inline src/paddle.cpp]: Same for boosted and slowed here. No need for ternary operator.

**Review by chesterhow — CHANGES_REQUESTED** [review]

File: src/paddle.cpp (hunk 2 above)
Comment (JeromeTiong) [reply src/paddle.cpp]: this is such that when the time is up it would set it back to false

Comment (chesterhow) [reply src/paddle.cpp]: That code does the same

Comment (JeromeTiong) [reply src/paddle.cpp]: nvm i got it

**Review by chesterhow — APPROVED** [review]

**Pull request merged**